mambaSharedFramework/Pantos-Generic HLS Playlist Parsing/Pantos-Generic Tag Validators/HLSPlaylistValidatorImpl.swift

```diff
@@ -86,7 +86,8 @@ public class HLSVariantPlaylistValidator: HLSExtensibleValidator {
                                                         HLSPlaylistRenditionGroupMatchingPROGRAM_IDValidator.self,
                                                         HLSPlaylistRenditionGroupMatchingNAMELANGUAGEValidator.self,
                                                         EXT_X_STARTTimeOffsetValidator.self,
-                                                        EXT_X_DATERANGEPlaylistValidator.self]
+                                                        EXT_X_DATERANGEPlaylistValidator.self,
+                                                        EXT_X_SESSION_DATAPlaylistValidator.self]
```

Comment:
Just realized I need to move this into the `HLSMasterPlaylistValidator.validators` array.